repl/src/main/scala/ammonite/repl/interp/Pressy.scala

```diff
@@ -97,15 +104,56 @@ object Pressy {
     }
 
     def scoped: (Int, List[(String, Option[String])]) = {
-      index -> ask(index, pressy.askScopeCompletion).map(s =>
-        pressy.ask(() => (s.sym.name.decoded, None))
-      )
+      pressy.ask { () =>
+        def isNotPackage(target: pressy.Tree) = target.symbol == null || target.symbol == pressy.NoSymbol || !target.symbol.hasPackageFlag
+        
+        val suggestions = tree.collect { 
+          case q"$target.$op" if target.pos.end <= index && isNotPackage(target) => (target, op, None)
+          case q"$target.$op($argument)" if target.pos.end <= index && isNotPackage(target) => (target, op, Some(argument))
+        }.sortBy(index - _._1.pos.end).headOption.flatMap {
+          case (operationTarget, operator, argument) =>
+            val completionInvocation = new String(operationTarget.pos.source.content.slice(operationTarget.pos.start, operationTarget.pos.end))
+            val completionIncovationArguments = Seq('"' + operator.decoded + '"', argument.map(a => new String(a.pos.source.content.slice(a.pos.start, a.pos.end))).getOrElse("()")).mkString(", ")
+            val wrapperName = "SuggesterWrapper" + eval.getCurrentLine + suggestionNumber.incrementAndGet
+            val code = s"""$previousImports
+                        |object $wrapperName {
+                        |def suggestions = ($completionInvocation).suggestions($completionIncovationArguments)
+                        |}""".stripMargin
+//            println(s"($completionInvocation).suggestions($completionIncovationArguments)")
```

Comment:
It is not good style to commit println and it is even worse to commit them as comments.